app-creativity/rnote/spec

```diff
@@ -0,0 +1,5 @@
+VER=0.11.0
+SRCS="tbl::https://github.com/flxzt/rnote/archive/v$VER.tar.gz"
+CHKSUMS="sha256::b133d4331963d3c09d3a7477f60fc4c5072471dcbf459379a593ca1724164af4"
+
+CHKUPDATE="github::repo=flxzt/rnote"
```

Comment:
Please use `anitya::` (I know, it's down right now, but let's try again in a few hours).